api/datalake_api/app.py

```diff
@@ -23,6 +23,7 @@
 from datalake_api import settings
 
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
This is already happening on line 39 with `DATALAKE_API_LOG_LEVEL` setting. It's probably fine to default to INFO this way, but i'm not sure if there are side effects from calling basicConfig more than once.